writers/influxdb/README.md

```diff
@@ -33,7 +33,7 @@ default values.
       MF_INFLUX_WRITER_PORT: [Service HTTP port]
       MF_INFLUX_WRITER_DB_NAME: [InfluxDB database name]
       MF_INFLUX_WRITER_DB_HOST: [InfluxDB database host]
-      MF_INFLUX_WRITER_DB_PORT: [InfluxDB port]
+      MF_INFLUX_WRITER_DB_PORT: [InfluxDB database port]
```

Comment:
`InfluxDB database port` remove database, that is already clear from `InfluxDB` name.